docs/hyperopt.md

```diff
@@ -150,6 +151,59 @@ The above setup expects to find ADX, RSI and Bollinger Bands in the populated in
 When you want to test an indicator that isn't used by the bot currently, remember to
 add it to the `populate_indicators()` method in `hyperopt.py`.
 
+## Loss-functions
+
+Each hyperparameter tuning requires a target. This is usually defined as a function, which get's closer to 0 for increasing values.
+
+By default, freqtrade uses a loss function we call `legacy` - since it's been with freqtrade since the beginning and optimizes for short trade duration.
+
+This can be configured by using the `--loss <value>` argument.
+Possible options are:
+
+* `legacy` - The default option, optimizing for short trades and few losses.
+* `sharpe` - using the sharpe-ratio to determine the quality of results
+* `custom` - Custom defined loss-function [see next section](#using-a-custom-loss-function)
+
+
+### Using a custom loss function
+
+To use a custom loss function, make sure that the function `hyperopt_loss_custom` is defined in your custom hyperopt class.
+You then need to add the command line parameter `--loss custom` to your hyperopt call so this fuction is being used.
+
+A sample of this can be found below.
+
+``` python
+    @staticmethod
+    def hyperopt_loss_custom(results: DataFrame, trade_count: int,
+                             min_date: datetime, max_date: datetime, *args, **kwargs) -> float:
+        """
+        Objective function, returns smaller number for more optimal results
+        """
+        total_profit = results.profit_percent.sum()
+        trade_duration = results.trade_duration.mean()
+
+        trade_loss = 1 - 0.25 * exp(-(trade_count - TARGET_TRADES) ** 2 / 10 ** 5.8)
+        profit_loss = max(0, 1 - total_profit / EXPECTED_MAX_PROFIT)
+        duration_loss = 0.4 * min(trade_duration / MAX_ACCEPTED_TRADE_DURATION, 1)
+        result = trade_loss + profit_loss + duration_loss
+        return result
+```
+
+Currently, the arguments are:
+
+* `results`: DataFrame containing the result
```

Comment:
"containing the results of backtesting for the hyperopt epoch".

And we need to describe the fields (columns) available.